conftest.py

```diff
@@ -12,41 +12,11 @@
 """
 
 import os
-import os.path as osp
```

Comment:
Please restore this import to fix our tests (it's used by `get_passed_tests` below).